src/Common/src/Interop/Windows/advapi32/Interop.OpenEventLog.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+
+internal partial class Interop
+{
+    internal partial class Advapi32
+    {
+        [DllImport(Libraries.Advapi32, CharSet = CharSet.Unicode, SetLastError = true)]
+        internal static extern SafeEventLogReadHandle OpenEventLog(string UNCServerName, string sourceName);
```

Comment:
Nit:
`UNCServerName` => `lpUNCServerName` to match the doc'd parameter name
`sourceName` => `lpSourceName` to match the doc'd parameter name